packages/server/modules/workspacesCore/domain/events.ts

```diff
@@ -7,7 +7,7 @@ export const workspaceEventNamespace = 'workspace' as const
 const eventPrefix = `${workspaceEventNamespace}.` as const
 
 export const WorkspaceEvents = {
-  Authorized: `${eventPrefix}authorized`,
+  Authorizing: `${eventPrefix}authorizing`,
```

Comment:
the previous name implied that the workspace was already authorized, so any errors thrown from this event handler wouldn't actually cancel the authorization